resources/rafter/templates/tests/test.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.tests.enabled }}
+{{- if .Capabilities.APIVersions.Has "testing.kyma-project.io/v1alpha1" }}
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    app: {{ .Chart.Name }}-tests
+    app.kubernetes.io/name: {{ .Chart.Name }}-tests
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  disableConcurrency: true
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+    spec:
+      {{ if .Values.global.isLocalEnv }}
```

Comment:
I'm not sure whether this section should stay, so I'm not introducting any changes in https://github.com/kyma-project/kyma/pull/6569/files#diff-e6cde1edab086956c4779fba43512d22R2-R12 for now - will do after review, after I'm sure